portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,7 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function changeBackgroundColor(color) {
+    document.body.style.background = color;
+}
```

Comment:
nit: Add a newline at the end of the file.  It's a convention (for some reason), but it's one we follow and also GitHub will yell about:

![image](https://user-images.githubusercontent.com/1651960/83548246-77029700-a4d1-11ea-91f8-39bbcb738b53.png)